src/api/procedures/modifyAllowance.ts

```diff
@@ -0,0 +1,144 @@
+import BigNumber from 'bignumber.js';
+import { TxTags } from 'polymesh-types/types';
```

Comment:
This should be imported from `~/types`. There are a couple of other places where this can be corrected